From: Li Zhang <lizhang@suse.de>
Date: Tue, 29 Mar 2022 12:04:16 +0200
Subject: Revert "python, iotests: replace qmp with aqmp"

References: bsc#1197528 bsc#1197150

aqmp is still not stable, it causes failures.
This reverts commit 76cd358671e6b8e7c435ec65b1c44200254514a9.

Signed-off-by: Li Zhang <lizhang@suse.de>
---
 python/qemu/machine/machine.py | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/python/qemu/machine/machine.py b/python/qemu/machine/machine.py
index a487c397459a1fa6889276ab2538..a0cf69786b4bb7e851b5eeb2517b 100644
--- a/python/qemu/machine/machine.py
+++ b/python/qemu/machine/machine.py
@@ -41,6 +41,7 @@ from typing import (
 )
 
 from qemu.qmp import (  # pylint: disable=import-error
+    QEMUMonitorProtocol,
     QMPMessage,
     QMPReturnValue,
     SocketAddrT,
@@ -49,12 +50,6 @@ from qemu.qmp import (  # pylint: disable=import-error
 from . import console_socket
 
 
-if os.environ.get('QEMU_PYTHON_LEGACY_QMP'):
-    from qemu.qmp import QEMUMonitorProtocol
-else:
-    from qemu.aqmp.legacy import QEMUMonitorProtocol
-
-
 LOG = logging.getLogger(__name__)
 
 
